Engine/ac/string.cpp

```diff
@@ -106,25 +108,38 @@ const char* String_Truncate(const char *thisString, int length) {
 const char* String_Trim(const char *thisString)
 {
     const auto &this_header = ScriptString::GetHeader(thisString);
-
     const char* nonSpaceFront = thisString;
-    for(int c = ugetc(nonSpaceFront); uisspace(c); )
-    {
-        nonSpaceFront += ucwidth(c);
-        c = ugetc(nonSpaceFront);
+    const char* end = thisString + this_header.Length;
+    const char* nonSpaceBack = end;
+
+    if(get_uformat() == U_UTF8) {
```

Comment:
One small request, could you add spaces between `if` and open bracket, similar to how it's done with `for`?
I noticed you often do `if(`, but that's inconsistent with the codestyle.

Reply:
I think VS does it automatically, so I am used to not care when using VS but I wrote that line in CLion, which doesn't do that.